lib/render/RayCaster.cpp

```diff
@@ -749,12 +908,10 @@ void RayCaster::_drawVolumeFaces( int            whichPass,
     else if( whichPass == 2 )
     {
         glUseProgram( _2ndPassShaderId );
-        uniformLocation = glGetUniformLocation( _2ndPassShaderId, "MVP" );
-        glUniformMatrix4fv( uniformLocation, 1, GL_FALSE, MVP );
-        uniformLocation = glGetUniformLocation( _2ndPassShaderId, "boxMin" );
-        glUniform3fv( uniformLocation, 1, _userCoordinates.boxMin );
-        uniformLocation = glGetUniformLocation( _2ndPassShaderId, "boxMax" );
-        glUniform3fv( uniformLocation, 1, _userCoordinates.boxMax );
+        GLint uniformLocation = glGetUniformLocation( _1stPassShaderId, "MV" );
```

Comment:
You are using the first pass shader id to configure the second pass shader

Reply:
this is addressed in 49d19d28.

Reply:
@StasJ will refactor to set uniforms using SHaderProgram class API